paymentapi/src/main/java/fi/hel/verkkokauppa/payment/api/data/PaymentMethodFilter.java

```diff
@@ -0,0 +1,31 @@
+package fi.hel.verkkokauppa.payment.api.data;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public enum PaymentMethodFilter {
+
+    // If namespace has multiple filters, all available methods for all will be returned
+    ORDER_TYPE(Collections.singletonList("asukaspysakointi")),
```

Comment:
If we have fixed filters (which means they can be referred to from properties easily with a single property), we should use env properties to select for whom they are active.